Makefile

```
@@ -211,6 +215,16 @@ bundle: kustomize operator-sdk ## Generate bundle manifests and metadata, then v
 	cd config/default && $(KUSTOMIZE) edit set image kube-rbac-proxy=$(KRP_IMAGE_BASE):$(KRP_IMAGE_VERSION)
 	if [ -n "$(IMAGE_PULL_SECRET_NAME)" ]; then cd config/default && $(KUSTOMIZE) edit add patch --kind Deployment --group apps --version v1 --name controller-manager --patch '${image_pull_secrets_patch}'; fi
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
+	@if [ -n "$(REPLACES)" ]; then \
+		echo "Adding replaces: $(REPLACES) to ClusterServiceVersion"; \
+		sed -i.bak -e '/^  provider:/i\' -e '  replaces: $(REPLACES)' bundle/manifests/nginx-ingress-operator.clusterserviceversion.yaml && \
+		rm -f bundle/manifests/nginx-ingress-operator.clusterserviceversion.yaml.bak; \
+	fi
+	@echo "Adding skips: [] to ClusterServiceVersion"; \
+	sed -i.bak -e '/^  version:/i\
```

```suggestion
	sed -i.bak '/^  version:/{h;s/.*/  skips: []/;p;x;}' bundle/manifests/nginx-ingress-operator.clusterserviceversion.yaml && \
```
same as above, and check if `make bundle` works with proper changes in clusterServiceVersion